lib/components/form/ViewInverters.tsx

```diff
@@ -0,0 +1,152 @@
+    error,
+    isLoading,
+  } = useSWR<Inverters>(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL_GET}/enode/inverters`
+  );
+
+  const { trigger } = useSWRMutation(
+    `${process.env.NEXT_PUBLIC_API_BASE_URL_POST}/sites/${siteUUID}/inverters`,
+    sendRequest
+  );
+
+  const [selectedInverters, setSelectedInverters] = useState<string[]>([]);
+  const [didSubmit, setDidSubmit] = useState(false);
+
+  const nextPageOrSubmit = async () => {
+    if (isSelectMode) {
+      if (!didSubmit) setDidSubmit(true);
+      await trigger({ client_ids: selectedInverters });
+    }
+    nextPageCallback();
+  };
+
+  const addOrRemove = (inverter: string) => {
+    if (selectedInverters.includes(inverter)) {
+      setSelectedInverters(selectedInverters.filter((e) => e !== inverter));
+    } else {
+      setSelectedInverters([...selectedInverters, inverter]);
+    }
+  };
+
+  // @TODO skeletons!!
+  return (
+    <div className="flex h-[var(--onboarding-height)] w-full flex-col items-center">
+      <div className="flex h-full w-11/12 max-w-lg flex-col justify-between md:mt-8 md:max-w-4xl">
+        <div className="flex w-full flex-grow flex-col p-3 pt-0">
+          {isSelectMode ? (
+            <>
+              <h1 className="mt-4 text-xl font-semibold text-white">
+                Select Inverters
+              </h1>
+              <h1 className="text-md mb-4 text-white">
+                Select the inverters that correspond to this site.
+              </h1>
+            </>
+          ) : (
+            <h1 className="mb-4 mt-4 text-xl font-semibold text-white">
+              Connected Inverters
+            </h1>
+          )}
+          <div className="mb-8 grid w-full grid-cols-1 items-center justify-center gap-4 md:mt-2 md:grid-cols-2">
+            {inverters?.inverters.map((inverter) => (
+              <InverterCard
+                inverter={inverter}
+                selectMode={isSelectMode}
+                selected={
+                  isSelectMode && selectedInverters.includes(inverter.id)
+                }
+                onClick={() => addOrRemove(inverter.id)}
+                key={inverter.id}
+              />
+            ))}
+          </div>
+          {!isSelectMode && (
+            <Link href="https://www.omfgdogs.com">
+              <a>
+                <div className="flex flex-row justify-center gap-2">
+                  <PlusCircleIcon width={24} height={24} color="#FFD053" />
+                  <h2 className="text-l text-ocf-yellow-500">
+                    Link more inverters
+                  </h2>
+                </div>
+              </a>
+            </Link>
+          )}
+          <Button
+            variant="solid"
+            disabled={isSelectMode && selectedInverters.length < 1}
+            hidden="md"
+            className="mb-8 mt-auto w-full self-center"
+            onClick={nextPageOrSubmit}
+          >
+            {isSelectMode ? 'Finish' : 'Next'}
+          </Button>
```

Comment:
Duplicating buttons on these pages seems to be the only way to do it with the differing styles and positioning... but the duplication on these files just makes them long and a bit harder to read. Maybe we can think about this later. Seems like the `hidden` attribute makes it cleaner at least.